project_euler/problem_49/sol1.py

```diff
@@ -0,0 +1,140 @@
+from itertools import permutations
+from math import floor, sqrt
+
```

Comment:
Oh, did not see that. Please put your imports after the module docstring.

Reply:
alright